src/fs/data/private/spiralratchet.test.ts

```diff
@@ -54,3 +92,31 @@ async function iterateAsync<T>(initial: T, f: (obj: T) => Promise<T>, n: number)
   }
   return obj
 }
+
+function ab2str(buf: ArrayBuffer): string {
+    var dataView = new DataView(buf);
+    var decoder = new TextDecoder('utf-8');
+    return decoder.decode(dataView);
+}
+
+function str2ab(str: string): ArrayBuffer {
+  var buf = new ArrayBuffer(str.length*2); // 2 bytes for each char
+  var bufView = new Uint16Array(buf);
+  for (var i=0, strLen=str.length; i < strLen; i++) {
+    bufView[i] = str.charCodeAt(i);
+  }
+  return buf;
+}
```

Comment:
```suggestion
function str2ab(str: string): ArrayBuffer {
  return new TextEncoder().encode(str).buffer
}
```
:grimacing: 
I'm not 100% sure. Was this function encoding into utf-16? Then my suggesting will do something different.
I'd propose to get rid of this anyway, by simply generating uint8arrays in fast-check and calling `.buffer` on those.
Leaving this here just FYI though :D